xaitk_saliency/interfaces/gen_object_detector_blackbox_sal.py

```diff
@@ -144,6 +147,10 @@ def generate(
             objectness,
         )
 
+        if len(output) == 0:
+            logging.info("No detections found for image. Check DetectImageObjects and saliency configuation")
```

Comment:
Do we think this qualifies as a warning (more than info)?

Reply:
When I talked with @bjrichardwebster about the issue, we both agreed we did not want to clutter the console with the logging info. I thought that (_by default_) warning would be output to the console. If that is wrong I have no issues with changing this.

Reply:
Okay, with that, lets keep this as `info` then. Most loggers are set to WARN and above